### PR TITLE
Home page and video listing

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (1147)
Screenshot (1148)
Screenshot (1149)

### CODE REVIEW

**Review by mehulsatardekar — APPROVED** [review]
Great work didn't find any issue 
appreciation: good work on proper naming CSS classes, and variable

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,23 @@
+import { useEffect } from "react";
 import "./App.css";
+import Navbar from "./Components/Navbar/Navbar";
+import { Routes, Route } from "react-router-dom";
+import Home from "./Pages/Home/Home";
+import LandingPage from "./Pages/Landing Page/LandingPage";
 
 function App() {
-  return <div className="App">React Video Lib</div>;
+  useEffect(() => {
```
Comment (Namrata-J) [inline src/App.js:9]: it's great that you are doing dom manipulations in useEffect.

File: src/Components/SideBar/SideBar.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  ];
+
+  return (
+    <div className="sideBar__container">
+      <ul className="sideBar__list">
+        {sideBarData.map((val, key) => {
+          return (
+            <li
+              key={key}
+              // id={window.location.pathname == val.link ? "active" : ""}
```
Comment (Namrata-J) [inline src/Components/SideBar/SideBar.js:45]: you can remove commented code from here

File: src/Pages/Home/CategoryList.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.trending__singleItem {
+  height: 50vh;
+  padding: 0.5rem;
+  margin: 0.5rem;
+  border-radius: 1rem;
+}
+
+.trending__singleItem {
+  -webkit-box-shadow: 3px 6px 22px 1px #086dd3;
+  -moz-box-shadow: 3px 6px 22px 1px #086dd3;
```
Comment (Namrata-J) [inline src/Pages/Home/CategoryList.css:10]: try using color variables

File: src/Pages/Home/Home.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
+.hero__section {
+  /* background-image: linear-gradient(160deg, #0f99e9 30%, #80d0c7 50%),
```
Comment (Namrata-J) [inline src/Pages/Home/Home.css:2]: remove commented code from here as well

File: src/Pages/Landing Page/LandingPage.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+import { useDispatch, useSelector } from "react-redux";
+import VideoCard from "../../Components/VideoCard/VideoCard";
+
+const LandingPage = () => {
+  const [categorySelected, setCategorySelected] = useState("All");
+
+  const dispatch = useDispatch();
+  const { data, status } = useSelector((state) => state.video);
+  // console.log(STATUSES);
+  // console.log(fetchingVideos());
```
Comment (Namrata-J) [inline src/Pages/Landing Page/LandingPage.js:14]: remove commented console, if not needed for future use

**Review by Namrata-J — APPROVED** [review]
Rest of your code looks good and modular!!

File: src/Components/SideBar/SideBar.js (hunk 2 above)
Comment (ashishkatiyar2404) [reply src/Components/SideBar/SideBar.js:45]: ok

File: src/Pages/Home/CategoryList.css (hunk 3 above)
Comment (ashishkatiyar2404) [reply src/Pages/Home/CategoryList.css:10]: yes, I will do later.

Comment (ashishkatiyar2404) [reply src/Pages/Home/CategoryList.css:10]: yes, I will do it later.

**Pull request merged**